tests/test_other.py

```diff
@@ -12167,15 +12167,4 @@ def test_rust_gxx_personality_v0(self):
           return 0;
         }
       }
-    ''')
-    self.run_process([EMXX, '-c', 'main.cpp', '-fexceptions'])
-
-    self.run_process([EMXX, '-o', 'main.js', 'main.o', '-fexceptions'] + self.get_emcc_args())
-
-    try:
-      self.do_run('main.js', no_build=True)
-      raise RuntimeError('should not have passed')
-    except AssertionError as e:
-      err = e
-    assert "Aborted(native code called abort())" in err.args[0]
-    del err
+    ''', 'Aborted(native code called abort())', assert_returncode=NON_ZERO)
```

Comment:
I don't think we need abort message here.. It's enough to ensure we have a nonzero return code.